pennylane_lightning/core/_version.py

```diff
@@ -16,4 +16,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.37.0-dev48"
+__version__ = "0.37.0-rc0"
```

Comment:
We need to update the version before the release.

Reply:
Done on the release branch `v0.37.0_release`.